src/shims/unix/linux/fd/socketpair.rs

```diff
@@ -0,0 +1,30 @@
+use crate::*;
+
+use crate::shims::unix::fs::FileDescriptor;
+
+use std::io;
+
+#[derive(Debug)]
+/// Pair of connected sockets
```

Comment:
```suggestion
/// Pair of connected sockets.
///
/// We currently don't allow sending any data through this pair, so this can be just a dummy.
```
Is that correct?